extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java

```diff
@@ -81,6 +81,15 @@ public static void addAllProjects(BufferedWriter wr) throws IOException {
 		write(wr, "maven { url \"" + DependencyBank.libGDXSnapshotsUrl + "\" }");
 		write(wr, "maven { url \"" + DependencyBank.libGDXReleaseUrl + "\" }");
 		write(wr, "}");
+		space(wr);
```

Comment:
I suggest adding a comment here with explanation of what this does, for future reference. It would also be nice to add such comment to the generated source, so that users know whether this task can be safely deleted/modified/etc.